External IDs Add (#1029)
spdx/spdx-3-model: model/Core/Vocabularies/ExternalIdentifierType.md

- cpe22: [Common Platform Enumeration Specification 2.2](https://cpe.mitre.org/files/cpe-specification_2.2.pdf)
- cpe23: [Common Platform Enumeration: Naming Specification Version 2.3](https://csrc.nist.gov/publications/detail/nistir/7695/final)
- cve: Common Vulnerabilities and Exposures identifiers, an identifier for a specific software flaw defined within the official CVE Dictionary and that conforms to the [CVE specification](https://csrc.nist.gov/glossary/term/cve_id).

> - duns: [The Dun & Bradstreet D‑U‑N‑S Number is a unique nine-digit identifier for businesses](https://www.dnb.com/en-us/smb/duns.html)

Reviewer: ```suggestion
- duns: [Data Universal Numbering System (D-U-N-S) Number](https://www.dnb.com/en-us/smb/duns.html) is a unique nine-digit identifier, issued by Dun & Bradstreet, that identifies a business entity, often on a location-specific basis.
```

To my understanding, DUNS number is meant to be location-specific (for example, a headquarters and a branch of the same business entity each of them may have its own DUNS number) -- although it is not required to be so. 

Do you think it is ok to have "often on a location-specific basis" at the end of the description? Or do you think it is unnecessary? Feel free to adjust.

> - duns: The Dun & Bradstreet D‑U‑N‑S Number is a unique nine-digit identifier for businesses. [D-U-N-S Number Questions: Start Here - Dun & Bradstreet](https://www.dnb.com/en-us/smb/duns.html)

Reviewer 2: ```suggestion
- duns: [The Dun & Bradstreet D‑U‑N‑S Number is a unique nine-digit identifier for businesses](https://www.dnb.com/en-us/smb/duns.html)
```

> - gln: [Global Location Number (GLN)](https://www.gs1.org/standards/id-keys/gln) can be used by companies to identify their locations

Reviewer: ```suggestion
- gln: [Global Location Number (GLN)](https://www.gs1.org/standards/id-keys/gln) is a 13-digit number, assigned by GS1, that uniquely identifies a legal entity (e.g., a company or customer), a function within a legal entity, a physical location (e.g., a warehouse or a specific shelf in a store), or a digital location (e.g., an Electronic Data Interchange (EDI) gateway).
```

Adapted from https://www.gs1.org/standards/id-keys/gln and https://en.wikipedia.org/wiki/Global_Location_Number

> - gln: Global Location Number (GLN) can be used by companies to identify their locations. [Global Location Number (GLN) | GS1](https://www.gs1.org/standards/id-keys/gln)

Reviewer 2: ```suggestion
- gln: [Global Location Number (GLN)](https://www.gs1.org/standards/id-keys/gln) can be used by companies to identify their locations
```

> - glue: [GLobal Unique Enterprise (GLUE) Identifiers](https://datatracker.ietf.org/doc/draft-ietf-spice-glue-id/)

Reviewer: ```suggestion
- glue: [GLobal Unique Enterprise (GLUE) Identifiers](https://datatracker.ietf.org/doc/draft-ietf-spice-glue-id/), as defined by the IETF Internet-Draft, is expressed as a GLUE URI, a Uniform Resource Identifier that standardizes the representation of existing organizational entity identifiers.
```

I'm trying to put some description here. Feel free to revise.

> - glue: GLobal Unique Enterprise (GLUE) Identifiers [GLobal Unique Enterprise (GLUE) Identifiers](https://datatracker.ietf.org/doc/draft-ietf-spice-glue-id/)

Reviewer 2: ```suggestion
- glue: [GLobal Unique Enterprise (GLUE) Identifiers](https://datatracker.ietf.org/doc/draft-ietf-spice-glue-id/)
```

> - gtin:  [Global Trade Item Number (GTIN)](https://www.gs1.org/standards/id-keys/gtin) can be used to uniquely identify all of its trade items

Reviewer: ```suggestion
- gtin: [Global Trade Item Number (GTIN)](https://www.gs1.org/standards/id-keys/gtin) is a number, assigned by GS1, that uniquely identifies a trade item (product or service).
```

In reference to [HashAlgorithm](https://spdx.github.io/spdx-spec/v3.0.1/model/Core/Vocabularies/HashAlgorithm/ ) vocabulary that differentiates the same algorithm with different digest length, do we like to have `gtin8`, `gtin12`, `gtin13`, and `gtin14` here instead?

An example from HashAlgorithm that have four entries of SHA3:

> sha3_224: SHA-3 with a digest length of 224, as defined in [FIPS 202](https://csrc.nist.gov/pubs/fips/202/final).
> sha3_256: SHA-3 with a digest length of 256, as defined in [FIPS 202](https://csrc.nist.gov/pubs/fips/202/final).
> sha3_384: SHA-3 with a digest length of 384, as defined in [FIPS 202](https://csrc.nist.gov/pubs/fips/202/final).
> sha3_512: SHA-3 with a digest length of 512, as defined in [FIPS 202](https://csrc.nist.gov/pubs/fips/202/final).

> - gtin: Global Trade Item Number (GTIN) can be used to uniquely identify all of its trade items [Global Trade Item Number (GTIN) | GS1](https://www.gs1.org/standards/id-keys/gtin)

Reviewer 2: ```suggestion
- gtin:  [Global Trade Item Number (GTIN)=](https://www.gs1.org/standards/id-keys/gtin) can be used to uniquely identify all of its trade items
```

> - hsCodes: The Harmonized System (HS) of tariff nomenclature is an internationally standardized system of names and numbers to classify traded products defined by [World Customs Organization](https://www.wcoomd.org/en/topics/nomenclature/overview/what-is-the-harmonized-system.aspx)

Reviewer: ```suggestion
- hsCodes: The [Harmonized System (HS)](https://www.wcoomd.org/en/topics/nomenclature/overview/what-is-the-harmonized-system.aspx) of tariff nomenclature is an internationally standardized system of names and numbers, defined by the World Customs Organization, used to classify traded products.
```

I reordered the "defined by ..." clause, so it clearly quantifies the "system", and not the "trade products"

> - hsCodes: The Harmonized System (HS) of tariff nomenclature is an internationally standardized system of names and numbers to classify traded products. [World Customs Organization](https://www.wcoomd.org/en/topics/nomenclature/overview/what-is-the-harmonized-system.aspx)

Reviewer 2: ```suggestion
- hsCodes: The Harmonized System (HS) of tariff nomenclature is an internationally standardized system of names and numbers to classify traded products defined by [World Customs Organization](https://www.wcoomd.org/en/topics/nomenclature/overview/what-is-the-harmonized-system.aspx)
```

> - lei: The Legal Entity Identifier (LEI) is a 20-character, alpha-numeric code based on the ISO 17442 standard developed by the International Organization for Standardization (ISO). [Identifying Organizations - the Legal Entity Identifier (LEI) - Organizational Identity – GLEIF](https://www.gleif.org/en/organizational-identity/introducing-the-legal-entity-identifier-lei)

Reviewer 2: ```suggestion
- lei: The [Legal Entity Identifier (LEI)](https://www.gleif.org/en/organizational-identity/introducing-the-legal-entity-identifier-lei) is a 20-character, alpha-numeric code based on the [ISO 17442](https://www.iso.org/standard/78829.html) standard developed by the International Organization for Standardization.
```
